docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+ atlantis plan -p terra-env-jcarlton
+```
+to which `broadbot` will reply:
+```text
+Ran Plan for project: terra-env-jcarlton dir: terra-env workspace: jcarlton
+
+Show Output
+```
+
+Following a successful plan, run `atlantis apply`, and if that's successful (and you get an approval), 
+you can merge the PR.
+## Step 2: Edit the Helm File and Auxiliary Files
+[Helm](https://helm.sh/) manages k8s packages called [Helm Charts](https://helm.sh/docs/topics/charts/),
+allowing large-scale Kubernetes deployments. These may be published in public or private repositories,
+similarly to Docker container applications. In turn, a [helmfile](https://github.com/roboll/helmfile)
+may be used to work with many charts in a large, heterogeneous application (like Terra).
+
+### Aside: look through the Charts
+The main Terra Helm charts don't need changing for our deployment, but we set up instructions to create
+an instance with it. Charts for Terra are located in the public [terra-helm repo](https://github.com/broadinstitute/terra-helm).
+The chart directory for workspace manager is [here](https://github.com/broadinstitute/terra-helm/tree/master/charts/workspacemanager).
+THe chart is actually surprisingly brief, showing only one dependency.
+```yaml
+apiVersion: v2
+name: workspacemanager
+version: 0.19.0
+description: Chart for Terra Workspace Manager
+type: application
+keywords:
+  - dsp
+  - broadinstitute
+  - terra
+  - workspacemanager
+sources:
+  - https://github.com/broadinstitute/terra-helm/tree/master/charts
+  - https://github.com/DataBiosphere/terra-workspace-manager
+dependencies:
+  - name: postgres
+    condition: postgres.enabled
+    version: 0.1.0
+    repository: https://broadinstitute.jfrog.io/broadinstitute/terra-helm-proxy
+```
+
+It's also possible to navigate the helm repo itself using the `helm` command:
+```shell script
+$ helm repo add terra-helm https://broadinstitute.github.io/terra-helm
+```
+```shell script
+$ helm repo update
+```
+```text
+...Successfully got an update from the "terra-helm" chart repository
+```
+
+Then just do 
+```shell script
+$ helm pull terra-helm/workspacemanager
+```
+and a .tgz file such as `workspacemanager-0.19.0.tgz` should be downloaded silently to the current directory.
+
+We don't need to make any changes to the `terra-helm` repo in order to set up a personal environment.
+
+### The Helmfile
+The helmfile allows specifying override values for a chart for a particular
+deployment, and the [terra-helmfile repo](https://github.com/broadinstitute/terra-helmfile)
+is the place providing and for editing those. This "gitOps" model
+is similar to the relationship between the private, Atlantis-enabled `terraform-ap-deployments` repo
+and the public `terraform-ap-modules` that we saw above.
+
+We don't actually need `helm` installed locally for this step, though it will be required later.
+
+
+Clone the [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) and look at the file structure.
```

Comment:
What happens if DevOps changes the file structure in the future? This may be a bit too much detail to have to keep in sync with the code.

Reply:
I don't know how much choice they have in the structure. I'm definitely pointing to the only things I know here, and it's not very deep at this point. 

The exact structure isn't terribly important to the narrative.